grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', function () {
+    // grunt addition for htmllint
+  });
+
+  grunt.registerMultiTask('l10n-json-to-html', 'Convert l10n JSON files to HTML', function () {
+    var done = this.async();
+    this.files.forEach(function (file) {
+      var content = '';
+      var src = file.src[0];
+      var path = src.split('/');
+      var locale = grunt.file.readJSON(src);
+      for (var val in locale) {
+        var value = locale[val];
+        if (typeof value !== 'object' && value !== '' && isNaN(value)) {
+          console.log(value);
+          content += value.toString() + '\n\n';
+          grunt.file.write('.tmp/' + path[1] + '/' + path[2] + '/' + path[3].replace('json', 'html'), content);
```

Comment:
Oops, I missed this... The `grunt.file.write()` should be OUTSIDE of the `for` loop. 
Otherwise we're overwriting the file on every single key, which is about 26,460 file writes (or roughly 26,260+/- more times than we want).